woocommerce/woocommerce-ios
# Product List: search results model (schema update in Storage layer)

Fixes #1286 

- [x] ⚠️ Update PR branch before merging

Update release notes:


## Changes
- Core Data schema change for model version 20
  - Added `ProductSearchResults` entity with `keyword: String` attribute and `products` relaitonship
  - Added `Product.searchResults` relationship
- Generated/regenerated Core Data subclass files
- Updated `MIGRATIONS.md`

## Testing
- Build app from `develop`, launch the app
- Build app from PR branch, launch the app, go to Products tab --> should work normally without crashing on Core Data migration
